src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -66,7 +66,8 @@ public PersistentDataStore() {
         String userName = (String) entity.getProperty("username");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         String password = (String) entity.getProperty("password");
-        User user = new User(uuid, userName, creationTime, password);
+        String about = (String) entity.getProperty("about");
```

Comment:
Do we know what happens for already existing Users? Will `entity.getProperty("about")` fail? Does it need to have a default fallback?

I think the behavior is to return the null value ("") for the property, but it would be good to double check and acknowledge if that's the behavior you would expect for existing users